src/utils/env.ts

```diff
@@ -21,11 +20,23 @@ const PC_ENV_TLDS = environments
         return tlds;
     }, [] as string[])
     .concat(__PC_DEV_ENVS__);
+const GC_DEV_EXTRA_ENVS = environments.concat(__GC_DEV_EXTRA_ENVS__ as Environment[]);
```

Comment:
if we can figure out a way to define the type in globals.d.ts, we won't need this type assertion here.

Reply:
maybe we name this `ALL_GC_ENVS`?  `GC_DEV_EXTRA_ENVS` reads like it's just the extra envs and through me off for a second.